docs/testimonials.md

```diff
@@ -0,0 +1,118 @@
+> improvement for auto-rebase &mdash; users no longer have to learn about old
+> versions of a revision unless they want to look at the obslog.
+>
+> Over and over, I'm struck by how well Jujutsu demonstrates this kind of
+> evolved thinking, which as an experienced version control developer I deeply
+> appreciate.
+
+&mdash; Rain, engineer at Oxide Computer Company, former VCS developer
+
+> Jujutsu is amazing... I couldn't have come up with anything remotely as
+> elegant.
+>
+> It's so rare that a solution attacks the innermost core of a problem so
+> thoroughly, I genuinely feel blessed to be in its presence. And also a bit
+> vindicated in not even trying to learn to use any of the tools that felt like
+> more crutches stacked upon a sand castle  
+
+&mdash; Anonymous user, speaking from the shadows
+
+> It's the easiest time I've ever had learning a tool this deeply this quickly, 
+> because of the ability to experiment and undo, instead of triple-checking 
+> before trying a new scary command.
+
+&mdash; Scott Olson, advanced Git user and now a Jujutsu user
+
+> I've started using jj for personal repos and it has quickly gone from "neat,
+> let's try this more" to "very neat, added to my permanent config and
+> automatically installed for new machines".
+
+&mdash; Poliorcetics, on GitHub
+
+> when I worked on the rust compiler, my job was to chain together a bunch of
+> strange and cursed tools that broke often. jujutsu breaks about half as
+> much, so that's pretty good i guess
+
+&mdash; jyn514, Rust contributor
+
+> Jujutsu is pretty cool, you can even keep most of your existing workflows 
+
+&mdash; Ben, who doesn't want you keeping your existing workflow
+
+> Wait, it's not called Jujitsu?
+
+&mdash; Phil, Mercurial contributor (who doesn't have to learn Git, now that
+Jujutsu exists)
+
+## What the developers have to say
+
+> I've been a FOSS contributor using Git for over 16 years, and Jujutsu
+> continues to amaze me every day. It has that sweet simplicity I was fond of in
+> Darcs, but it boils down all my most core and fundamental workflows &mdash;
+> developed over years of experience &mdash; into a simple set of primitives.
+> The internal design is simple and beautiful; it looks like a database, making
+> the implementation elegant, safe, and extensible. All this, using the same Git
+> repositories my coworkers use.
+>
+> It's like if you found out one day that you built your entire home on a vein
+> of rich gold. Every day I seem to find new and beautiful emergent behaviors,
+> all adding up to a tool that is greater than the sum of its parts.
+
+&mdash; Austin Seipp, "No 1. Jujutsu Fan"
+
+> Honestly, I implemented signing support mostly for that sweet dopamine hit
+> that you get from the green checkmark on GitHub. Yeah.
+
+&mdash; Anton Bulakh, contributor and dopamine enthusiast
+
+> I'm sometimes still surprised that navigating with `next` and `prev` works.
```

Comment:
```suggestion
> I'm sometimes still surprised that navigating with `jj next` and `jj prev` works.
```

should probably be symmetric with the punchline

Reply:
done